data/mods/more_classes_scenarios/cs_classes.json

```diff
@@ -66,60 +66,6 @@
       "female": [ "sports_bra", "boxer_shorts" ]
     }
   },
-  {
```

Comment:
Was it an intended removal?

Reply:
Yes, it's in an obsoleted mod and incorporates obsoleted content, no need to keep it around.